app/actions/remote/channel.ts

```diff
@@ -194,16 +196,19 @@ export const fetchMyChannelsForTeam = async (serverUrl: string, teamId: string,
     }
 
     try {
-        let [channels, memberships]: [Channel[], ChannelMembership[]] = await Promise.all([
+        // eslint-disable-next-line prefer-const
+        let [channels, memberships, categoriesWithOrder]: [Channel[], ChannelMembership[], CategoriesWithOrder] = await Promise.all([
```

Comment:
Is it possible to remove the type definition? (client functions should be typed already)